src/BroadcasttClient.php

```diff
@@ -527,15 +527,15 @@ public function presenceAuth($channel, $socketId, $userId, $userInfo = null)
      *
      * @param $cluster
      */
-    public function useCluster($cluster)
+    public function useCluster($cluster): void
```

Comment:
```suggestion
    public function useCluster(string $cluster): void
```

PHPDoc has to be fixed as well